rest-api/src/main/java/uk/gov/hmcts/ccd/definition/store/rest/endpoint/DisplayApiController.java

```diff
@@ -126,7 +126,7 @@ public WorkBasketResult displayWorkBasketDefinitionIdGet(
         @ApiResponse(code = 200, message = "Unexpected error")
     })
     public SearchCasesResult displaySearchCasesResultIdGet(
-        @ApiParam(value = "Case Type ID", required = true) @PathVariable("id") String id, @RequestParam(value = "usecase", required = false) String usecase) {
+        @ApiParam(value = "Case Type ID", required = true) @PathVariable("id") String id, @RequestParam(value = "use_case", required = false) String usecase) {
```

Comment:
Since it is now considered two words:

```suggestion
        @ApiParam(value = "Case Type ID", required = true) @PathVariable("id") String id, @RequestParam(value = "use_case", required = false) String useCase) {
```